IoTVisualizer/src/IoTExtension/Utils.ts

```diff
@@ -159,9 +164,46 @@ export const removeConsumerCallback = (key: string) => {
   });
 };
 
+export const resolveAlert = async (iotId: any, connectionId: any) => {
+  const data = { deviceId: iotId, connectionStringId: connectionId, action: "resolve" };
+  const response = await fetch(`${process.env.IMJS_FUNCTION_APP_URL}/c2d-simulator` ?? "", {
+    method: "POST",
+    body: JSON.stringify(data),
+  }).catch((error) => console.log(`Request failed: ${error}`));
+  if (response && response.status === 200) {
+    return true;
+  }
+  return false;
+};
+
+export const func = async (elementId: any) => {
+  const temp = resolveAlert(elementId, "IOT_HUB_CONNECTION_STRING2");
+  return temp;
+};
+
+function getLabel(myArray: any[], iotId: any) {
+  let label = "";
+  myArray.forEach(function (element: any) {
+    if (element.iotId === iotId) {
+      label = element.label;
+    }
+  });
+  return label;
+}
+
+function getLoc(myArray: any[], iotId: any) {
+  let loc: Point3d = Point3d.create(0, 0, 0);
+  myArray.forEach(function (element: any) {
+    if (element.iotId === iotId) {
+      loc = element.origin;
+    }
+  });
+  return loc;
+}
+
 export const getDeviceDataFromTelemetry = (msg: any, deviceList?: SmartDevice[]) => {
 
-  const data: { iotId: string, value: string, unit: string, phenomenon: string, timeStamp: Date }[] = []; // Todo: can Queue data str. be used??
+  const data: { iotId: string, value: string, unit: string, phenomenon: string, timeStamp: Date, isChecked: boolean }[] = []; // Todo: can Queue data str. be used??

```

Comment:
What this isChecked boolean is doing? If we disable alerting feature and even though pass this boolean, how would that affect?